<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>626</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="1" column="0">
     <widget class="QFrame" name="frameNav">
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="buttonHome">
         <property name="toolTip">
          <string>Navigates to the home page</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset theme="QIcon::ThemeIcon::GoHome"/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonPlayback">
         <property name="toolTip">
          <string>Navigates to the playback page</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset theme="QIcon::ThemeIcon::Computer"/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QStackedWidget" name="pagesStack"/>
    </item>
    <item row="0" column="0">
     <widget class="CustomTitleBar" name="titleBar" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>626</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
    <addaction name="actionRestart"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionCameraProperties"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionQuick_Start_Guide"/>
   </widget>
   <widget class="QMenu" name="menuVideo">
    <property name="title">
     <string>Video</string>
    </property>
    <addaction name="actionRecord"/>
    <addaction name="actionOpen_Output_Directory"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionGitHub"/>
    <addaction name="actionCredits"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuVideo"/>
   <addaction name="menuOptions"/>
   <addaction name="menuAbout"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionQuit">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::SystemShutdown"/>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionRestart">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::SystemReboot"/>
   </property>
   <property name="text">
    <string>Restart</string>
   </property>
  </action>
  <action name="actionRecord">
   <property name="text">
    <string>Record</string>
   </property>
  </action>
  <action name="actionOpen_Output_Directory">
   <property name="text">
    <string>Open Output Directory</string>
   </property>
  </action>
  <action name="actionGitHub">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::InsertLink"/>
   </property>
   <property name="text">
    <string>GitHub</string>
   </property>
  </action>
  <action name="actionCredits">
   <property name="text">
    <string>Credits</string>
   </property>
  </action>
  <action name="actionQuick_Start_Guide">
   <property name="text">
    <string>Quick Start Guide</string>
   </property>
  </action>
  <action name="actionCameraProperties">
   <property name="text">
    <string>Camera Properties</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CustomTitleBar</class>
   <extends>QWidget</extends>
   <header>widgets/CustomTitleBar/customtitlebar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../MainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
